### PR TITLE
Build assets inside the container

### PR DESCRIPTION
**Pull request by dvaldivia**
This makes it so the dockerfile builds the assets inside of it before compiling the final version of the container.

Also addresses #183

### CODE REVIEW

**Review by Alevsk — APPROVED** [review]

**Review by bexsoft — CHANGES_REQUESTED** [review]
Please review an issue related with. seems like a socket is failing, I tried to re-execute flow and it didn't work
```
2020/10/29 16:35:44 error unexpected CloseError on ReadMessage: websocket: close 1006 (abnormal closure)
2020/10/29 16:35:44 error on ReadMessage: error
2020/10/29 16:35:44 closed conn.ReadMessage: websocket: close 1001 (going away)
--- PASS: TestWSHandle (0.00s)
FAIL
FAIL	github.com/minio/console/restapi	12.961s
?   	github.com/minio/console/restapi/operations	[no test files]
?   	github.com/minio/console/restapi/operations/admin_api	[no test files]
?   	github.com/minio/console/restapi/operations/user_api	[no test files]
FAIL
Makefile:53: recipe for target 'test' failed
make: *** [test] Error 1
Error: Process completed with exit code 2.```

**Comment by dvaldivia** [comment]
@bexsoft the strange thing is that I didn't change code D: only updated the Go version

**Review by Alevsk — DISMISSED** [review]

**Review by bexsoft — DISMISSED** [review]

**Review by Alevsk — DISMISSED** [review]

**Review by harshavardhana — APPROVED** [review]

File: .goreleaser.yml [hunk 1]
```diff
@@ -103,7 +107,7 @@ dockers:
     goos: linux
     # GOARCH of the built binary that should be used.
     goarch: amd64
-    dockerfile: Dockerfile.release
+    dockerfile: d
```
Comment (harshavardhana) [inline .goreleaser.yml:110]: is this correct? @dvaldivia

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: .goreleaser.yml (hunk 1 above)
Comment (dvaldivia) [reply .goreleaser.yml:110]: nein, that looks like an ugly mistake

Comment (dvaldivia) [reply .goreleaser.yml]: fixed

**Review by Alevsk — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**